src/index.html

```diff
@@ -8,6 +8,58 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars--0 container" >
+      <div class="star---image"></div>
+      <div class="star---image"></div>
+      <div class="star---image"></div>
+      <div class="star---image"></div>
+      <div class="star---image"></div>
+    </div>
+
+    <div class="stars--1 container">
+      <div class="star---image line---001"></div>
```

Comment:
```suggestion
      <div class="star-image"></div>
```
Do you really need this modificator line-001? You can use your container name for this task

Reply:
спробувала, але так в мене не працює